src/components/Auth/OAuthSection/OAuthSection.spec.tsx

```
@@ -0,0 +1,156 @@
+import { OAuthSection } from './OAuthSection';
+import { vi, describe, it, expect, afterEach } from 'vitest';
+
+const mockOAuthProviders = vi.hoisted(() => ({
+  GOOGLE: { enabled: true },
+  GITHUB: { enabled: true },
+}));
+
+// Mock child OAuth buttons to keep this a unit test
+vi.mock('../OAuthButton/GoogleOAuthButton', () => ({
+  default: ({ mode }: { mode: string }) => (
+    <button data-testid="google-oauth-button" data-mode={mode}>
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)